app/models/item.rb

```diff
@@ -1,3 +1,10 @@
 class Item < ApplicationRecord
+  validates_presence_of :name
+  validates_presence_of :description
+  validates_presence_of :unit_price
+  validates_presence_of :enabled
```

Comment:
Follow same formatting: `validates :attribute_name, presence: true`

Reply:
Add validation for boolean value:
`validates :enabled, inclusion: { in: [true, false] }`